src/inverseRobot.js

```diff
@@ -7,7 +7,20 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const KEYS = Object.values(robot);
```

Comment:
The variable name 'KEYS' is not descriptive enough. It's better to use a name that reflects the purpose of the variable. For example, 'robotValues' would be a more appropriate name.